Assets/ObstacleTower/Scripts/AgentLogic/ObstacleTowerAgent.cs

```diff
@@ -112,7 +111,7 @@ private void CompletedLevel()
 
     private void OnCollisionEnter(Collision col)
     {
-        if (IsDone()) return;
+        // if (IsDone()) return;
```

Comment:
Should this be commented out? I assume this is because being done is no longer possible.